pages/faq/troubleshooting/troubleshooting.md

```diff
@@ -94,6 +95,25 @@ Raspberry Pi 4 Model B Rev 1.5
 
 For more information about Raspberry Pi 4 Model B Rev 1.5, please refer to the [Product Information Portal](https://pip.raspberrypi.com/restricted) by Raspberry Pi.
 
+### balenaOS boot issues with Rev 1.1 of Raspberry Pi Compute Module 4 hardware
+
+Balena supports the use of the Raspberry Pi Compute Module 4 (CM4) on the Raspberry Pi CM4 IO Board. [Raspberry Pi](https://www.raspberrypi.com/) recently released [hardware Rev 1.1](https://forums.raspberrypi.com/viewtopic.php?t=337023) of the CM4. This revision of the module requires [balenaOS](https://balena.io/os) v2.91.1 or later. Earlier versions of balenaOS will not boot.
+
+To determine if you have hardware Rev 1.1 of the CM4, follow either of the steps below:
+
+1. Run the following command in balenaOS v2.91.1 or later. The output should confirm the hardware revision.
```

Comment:
Technically they could run this command on Raspberry Pi OS or Ubuntu or similar to learn what rev they have before trying balenaOS? If they are already running balenaOS on this hardware this probably doesn't apply to them.

Reply:
:chicken: :egg: 

:wink:

Reply:
Good one @lurch but agree with @klutchell's suggestion. You probably won't be able to boot into balenaOS if you are using the wrong rev. So generalized instructions should be the way to go. Or actually, just give them the hardware way of checking the rev first.

Reply:
Updated to clarify how to identify CM4 v1.1. Put visual method first. Then a method using Raspberry Pi OS.